### PR TITLE
Milestone: Feature Branch

### PR DESCRIPTION
**Pull request by yaseerokino**
## In this pull request i:

- added html&css linters
- added .html, .css and test.md files

### CODE REVIEW

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
# STATUS: INVALID :x::x:
Hi @yaseerokino 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. You've done well implementing some features that were asked in the project requirements, however, I will mark this project as invalid because you didn't follow the requirements, please work on it and ask for another review.

#### :warning: Checkout my comment on this: :writing_hand:

- [x] Your commits look unprofessional, please take a look at this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) to learn how to make descriptive commits or how to make sure that my repo is professional?

- [ ] From your PR, it seems like you have conflicts you need to resolve first as well as checking if the linters are passing successful.

readme



- [ ] Kindly read this, the README file: This is the first file one should read when starting a new project. It’s a set of useful information about a project and a kind of manual. It is the first file GitHub or any Git hosting site will show when someone opens your repository.

Please use this [template](https://github.com/microverseinc/readme-template) for more explanation. :white_check_mark:

 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by yaseerokino** [comment]
- I took a look at writing descriptive commits and I understand the errors I made.
- I was able to resolve the conflicts and fixed the StyleLint errors I had.
- Additionally, I made an error on my last commit message which states "fixed block-opening-space-before error" instead of "Fixed block-opening-space-before Stylint error"

File: README.md [hunk 1]
```diff
@@ -13,23 +13,14 @@ Additional description about the project and its features.
...
 
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
```
Comment (mimipeshy) [inline README.md:16]: - [ ] Kindly add the instructions needed to set up the project to make the section complete

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# ignore test.md
+
+test.md
```
Comment (mimipeshy) [inline .gitignore:3]: - [ ] Kindly also add node_modules as part of .gitignore as it can be added by accident while setting up linters

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @yaseerokino 👋🏾 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
- Great git workflow ✔️ 
- Great PR summary ✔️ 
- Good job on setting up linters ✔️ 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
------


https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/

**Comment by yaseerokino** [comment]
- I updated my README.md for both my main and feature branch to avoid any conflicts
- I also added node-modules to the .gitignore file

**Review by tahiry-dev — APPROVED** [review]
Hello @yaseerokino 👋, and welcome to Microverse! 

Congratulations! 🎉 , Your project is complete!  You have created a professional readme file  and you ignored the MIT.md file as suggested by the previous TSE💪💪

You can now merge it and move to the next step!, 
 👉keep up the good work

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**